pydantic/_internal/_generate_schema.py

```diff
@@ -473,26 +464,10 @@ def _generate_schema(self, obj: Any) -> core_schema.CoreSchema:  # noqa: C901
         elif issubclass(origin, typing.Tuple):  # type: ignore[arg-type]
             # TODO: To support generic subclasses of typing.Tuple, we need to better-introspect the args to origin
```

Comment:
Should this comment be removed? Not sure how you feel about generic subclasses of Tuple in the context of this PR